catalogs/catalog-jdbc-doris/src/main/java/com/datastrato/gravitino/catalog/doris/operation/DorisTableOperations.java

```diff
@@ -194,6 +191,65 @@ static void appendIndexesSql(
     sqlBuilder.append(",").append(NEW_LINE).append(indexSql);
   }
 
+  private static void appendPartitionSql(
+      Transform[] partitioning, JdbcColumn[] columns, StringBuilder sqlBuilder) {
+    if (partitioning == null || partitioning.length == 0) {
+      return;
+    }
+
+    StringBuilder partitionSqlBuilder = new StringBuilder();
+    Set<String> columnNames =
+        Arrays.stream(columns).map(JdbcColumn::name).collect(Collectors.toSet());
+
+    if (partitioning[0] instanceof Transforms.RangeTransform) {
```

Comment:
@zhoukangcn 
Could you help to review this PR if you have time? I see no problem here.